spec/models/tempr_spec.rb

```diff
@@ -88,6 +88,23 @@
       it { expect(tempr.endpoint_type).to eq('http') }
     end
   end
+
+  describe '#destroying' do
```

Comment:
so the syntax `describe #destroying` is used for a method name # is for instance methods and :: is for class methods. Good website for this stuff here - https://www.betterspecs.org

Since you're testing the destroy method here, it should be `describe #destroy`